src/Compatibility/Deprecated/ResampleITK::VectorImage/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 3.10.2)
+
+project(ResampleITK::VectorImage)
```

Comment:
is `::` problematic for project names?

Reply:
Fixed in several places. 

Purged all uses of ResampleITK::Vector to be replaced by ResampleITKVectorImage in a manner similar to other examples.